packages/kit/src/provider/Container/InAppNotification/index.tsx

```
@@ -1,21 +1,44 @@
+import {
+  ESwapApproveTransactionStatus,
+  ESwapSource,
+} from '@onekeyhq/shared/types/swap/types';
 
 import backgroundApiProxy from '../../../background/instance/backgroundApiProxy';
 import { AccountSelectorProviderMirror } from '../../../components/AccountSelector';
 import { useActiveAccount } from '../../../states/jotai/contexts/accountSelector';
+import { handleSwapNavigation } from '../../../views/Swap/hooks/useSwapNavigation';
+import {
+  appEventBus,
+  EAppEventBusNames,
```

ESLint

[error] Member 'EAppEventBusNames' of the import declaration should be sorted alphabetically.

(sort-imports)